website/src/_includes/section-menu/contribute.html

```diff
@@ -12,20 +12,33 @@
 
 <li><span class="section-nav-list-main-title">Contribute</span></li>
 <li><a href="{{ site.baseurl }}/contribute/">Get started contributing</a></li>
-<li><a href="https://cwiki.apache.org/confluence/display/BEAM/Using+IntelliJ+IDE">Using IntelliJ IDE</a></li>
+<li><a href="https://cwiki.apache.org/confluence/display/BEAM/Development+Environment+FAQ">FAQ</a></li>
 <li>
   <span class="section-nav-list-title">Technical Docs</span>
 
   <ul class="section-nav-list">
+    <li><a href="{{ site.baseurl }}/roadmap/">Roadmap</a></li>
     <li><a href="{{ site.baseurl }}/contribute/testing/">Testing guide</a></li>
-    <ul>
-      <li><a href="{{ site.baseurl }}/contribute/precommit-triage-guide/">Pre-commit Slowness Triage Guide</a></li>
-    </ul>
+    <li><a href="{{ site.baseurl }}/contribute/precommit-triage-guide/">Pre-commit Slowness Triage</a></li>
     <li><a href="{{ site.baseurl }}/contribute/ptransform-style-guide/">PTransform style guide</a></li>
     <li><a href="{{ site.baseurl }}/contribute/runner-guide/">Runner authoring guide</a></li>
     <li><a href="{{ site.baseurl }}/contribute/docker-images/">Docker images</a></li>
     <li><a href="{{ site.baseurl }}/contribute/design-documents/">Design documents</a></li>
     <li><a href="{{ site.baseurl }}/contribute/dependencies/">Dependencies guide</a></li>
+    <li><a href="{{ site.baseurl }}/contribute/feature-branches/">Feature branches</a></li>
+  </ul>
+</li>
+<li>
+  <span class="section-nav-list-title">Development Environment</span>
+  <ul class="section-nav-list">
+    <li><a href="https://cwiki.apache.org/confluence/display/BEAM/Using+IntelliJ+IDE">Using IntelliJ IDE</a></li>
```

Comment:
I feel like this list will get stale quickly-- links will get moved/renamed without updating here (I already did that with IntelliJ), or people will add new Tips pages without also adding here.

I don't mind too much as long as we don't expect this list to always match what's on the wiki. I'd be happy to just keep our "most important" tips pages linked here, for some measure of "most important"

It would be cool if there was a Jekyll plugin to automatically come up with a link roll from Confluence during the build. From a quick Google search I don't see such a thing.

Reply:
This list should be removed from here, since it is already on the wiki left nav: 

https://cwiki.apache.org/confluence/display/BEAM/Development+Environment+FAQ

(links could be repeated on the Wiki FAQ page as well)

Reply:
Removed the list